robot/EDA/tests/browser/health_check/eda_health_check.robot

```diff
@@ -0,0 +1,43 @@
+*** Settings ***
+Documentation   Validate health check for EDA
+Resource        robot/EDA/resources/EDA.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/EDA/resources/SettingsHealthCheckPageObject.py
+
+Suite Setup     Open Test Browser
+Suite Teardown  Capture screenshot and delete records and close browser
+
+*** Test Cases ***
+Verify settings health check app is displayed
+    [Documentation]         Validates 'Settings Health Check" app is displayed under app tile
+    [tags]                  unstable        rbt:high        W-8864633
+    Open app launcher
+    Verify item exists       Settings Health Check
+
+Verify last run date is updated upon clicking run health check Button
+    [Documentation]         Validates the last run date is updated accordingly after clicking the
+    ...                     run health check button.
+    [tags]                  unstable        rbt:high        W-8874163
+    Reload Page
+    Select App Launcher Tab      Settings Health Check
+    Current page should be      Home        Settings Health Check
+    Click health check button       Run Health Check
+    Verify last run date
+    Go to custom settings setup
+    Select App Launcher Tab      Settings Health Check
+    Verify last run date
+
+Verify account model health check settings checks pass
+    [Documentation]         Validates the account model health card is displayed with settings for
+    ...                     Administrative, Household, Default Account Model and Unique account
+    ...                     record type checks.Also verifies the status of each setting.
+    [tags]                  unstable        rbt:high        W-8880346
+    Click health check button       Run Health Check
```

Comment:
Start with a keyword that navigates to the health check page. Because if your 2nd test fails, this will fail too.

Reply:
I wanted to create dependency between tests and thats why I avoided launching app for every tests. I thought of using robotframework-dependencylibrary but then after discussing with Bryan decided to wait until next robot release as this feature will be available. As a temporary fix, I have leverage STATUS of previous test case to create dependency